util/testutil/integration/sandbox.go

```diff
@@ -115,13 +126,26 @@ func newSandbox(ctx context.Context, w Worker, mirror string, mv matrixValue) (s
 	}
 	deferF.Append(closer)
 
+	home, err := os.CreateTemp("", "buildkit-integration-home")
```

Comment:
as this is just `TempDir` maybe better to expose that directly. Not sure how it differs from `t.TempDir` though.